src/browser/components/DAppContainer/DAppContainer.scss

```diff
@@ -1,3 +1,8 @@
 .dAppContainer {
   height: 100%;
+  display: none;
```

Comment:
This isn't really a fix to the reload, right? How does a regular browser handle this?

Reply:
It is a fix because we're now just hiding the DOM element instead of allowing React to unmount its host component.